### PR TITLE
WRQ-23782: Add missing ui/ss tests Panels --> WizardPanels

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added missing ui and screenshot tests

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Newly UI tests added: 
- Picker with `joined` prop
- PopupTabLayout with various Items
- Scroller with focus outside container
- WizardPanels navigating with 5-way

Newly SS tests added:
- Panels with opaque back button
- PopupTabLayout with `scrimType` translucent and transparent

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Fixed various typos

### Links
[//]: # (Related issues, references)
WRQ-23782

### Comments
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

**Review by ion-andrusciac-lgp — APPROVED** [review]
LGTM👍

File: tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWTC-2008-specs.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+describe('qa sampler', function () {
+	const {popupTabLayout} = Page.components;
+
+	beforeEach(async function () {
+		await Page.open('WithVariousItems');
+	});
+
+	describe('PopupTabLayout', function () {
+		it('should navigate with left and right key [QWTC-2008]', async function () {
```
Comment (daniel-stoian-lgp) [inline tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWTC-2008-specs.js:12]: why did you add QWTC-2008 ? It  is  a test case for VirtualList

Comment (adrian-cocoara-lgp) [reply tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWTC-2008-specs.js:12]: Yes, I automated some existing test cases and these are their IDs

File: tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWT-2008-specs.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+describe('qa sampler', function () {
+	const {popupTabLayout} = Page.components;
+
+	beforeEach(async function () {
+		await Page.open('WithVariousItems');
+	});
+
+	describe('PopupTabLayout', function () {
+		it('should navigate with left and right key [QWT-2008]', async function () {
```
Comment (dongsuwon) [inline tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWT-2008-specs.js:12]: Is it right test case number? I cannot find matched test case.

File: tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWZ-5789-specs.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+describe('qa-sampler', function () {
+	const {popupTabLayout} = Page.components;
+
+	beforeEach(async function () {
+		await Page.open('WithVariousItems', '?locale=ar-SA');
+	});
+
+	describe('PopupTabLayout', function () {
+		it('should not navigate with left and right key in RTL - [QWZ-5789]', async function () {
```
Comment (dongsuwon) [inline tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWZ-5789-specs.js:12]: we don't use QWZ project.
```suggestion
		it('should not navigate with left and right key in RTL - [QWTC-2649]', async function () {
```

File: tests/ui/specs/WizardPanels/WizardPanels-specs.js [hunk 4]
```diff
@@ -186,5 +186,26 @@ describe('WizardPanels', function () {
 
 			expect(actual).toBe(expected);
 		});
+
+		it('should focus buttons when navigating with 5-way - [QWZ-5190]', async function () {
```
Comment (dongsuwon) [inline tests/ui/specs/WizardPanels/WizardPanels-specs.js:190]:
```suggestion
		it('should focus buttons when navigating with 5-way - [QWTC-2012]', async function () {
```

File: tests/ui/specs/Scroller/WithFocusOutsideContainer/Scroller-QWT-2652-specs.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+			expect(await focusToMeButton.isFocused()).toBe(true);
+		});
+	});
+
+	describe('5-way RTL', function () {
+		beforeEach(async function () {
+			await ScrollerPage.open('WithFocusOutsideContainer', '?locale=ar-SA');
+		});
+
+		it('should spotlight an outside of container element with \'native\' scrollMode', async function () {
```
Comment (dongsuwon) [inline tests/ui/specs/Scroller/WithFocusOutsideContainer/Scroller-QWT-2652-specs.js:24]:
```suggestion
		it('should spotlight an outside of container element with native scrollMode', async function () {
```

File: tests/ui/specs/Scroller/WithFocusOutsideContainer/Scroller-QWT-2652-specs.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+const ScrollerPage = require('../ScrollerPage');
+
+describe('Scroller', function () {
+	describe('5-way LTR', function () {
+		beforeEach(async function () {
+			await ScrollerPage.open('WithFocusOutsideContainer');
+		});
+
+		it('should spotlight an outside of container element with \'native\' scrollMode [QWT-2652]', async function () {
```
Comment (dongsuwon) [inline tests/ui/specs/Scroller/WithFocusOutsideContainer/Scroller-QWT-2652-specs.js:9]: QWT project also archived.
```suggestion
		it('should spotlight an outside of container element with native scrollMode [QWTC-1998]', async function () {
```

**Review by dongsuwon — CHANGES_REQUESTED** [review]

File: tests/screenshot/apps/components/PopupTabLayout.js [hunk 7]
```diff
@@ -87,6 +87,14 @@ const PopupTabLayoutTests = [
 		component: <PopupTabLayout open scrimType="transparent">{tabs}</PopupTabLayout>,
 		wrapper: {full: true}
 	},
+	{
+		component: <PopupTabLayout open scrimType="translucent">{tabs}</PopupTabLayout>,
```
[the remark this answers is not in the document]
Comment (adrian-cocoara-lgp) [reply tests/screenshot/apps/components/PopupTabLayout.js:91]: you're right. I deleted this test

**Comment by dongsuwon** [comment]
LGTM

**Review by dongsuwon — APPROVED** [review]
